safe_transaction_service/history/models.py

```diff
@@ -1238,8 +1238,8 @@ class MultisigTransaction(TimeStampedModel):
     signatures = models.BinaryField(null=True, blank=True)  # When tx is executed
     nonce = Uint256Field(db_index=True)
     failed = models.BooleanField(null=True, blank=True, default=None, db_index=True)
-    origin = models.CharField(
-        null=True, blank=True, default=None, max_length=200
+    origin = models.JSONField(
+        null=True, blank=True, default=None
```

Comment:
@Uxio0 What do you think about this change? In our staging database there are Null and random strings.

Reply:
Random strings are valid jsons, but null are not. I would make `{}` the new `None`, so I won't allow `null=True` and we need to run a migration before so every `None` becomes `{}` (I don't know if Django can handle that)

Reply:
`None` is converted to `{}` if default is `dict`, but we have an issue migrating the strings without `{` like following:
Before migration
```
 {'origin': '{"url": "https://example2.com", "name": "afeApp2"}'},
 {'origin': 'eoo'},
 {'origin': None}]
```
When try to migrate the following error is showed:
```
django.db.utils.DataError: invalid input syntax for type json
DETAIL:  Token "eoo" is invalid.
CONTEXT:  JSON data, line 1: eoo
```
I can fix it doing converting `'` to `"` :
```
In : tx.origin
Out: 'eoo'
In: tx.origin = "eoo"
In: txs.save()
```
@Uxio0 this could be handle in the migration but maybe is going to be a heavy migration, what do you think?

Reply:
Done here https://github.com/safe-global/safe-transaction-service/pull/1177/commits/923a47b3697580c4ea5556cd662f8e92c26ac4a5